docs/astro.config.mts

```diff
@@ -170,15 +174,29 @@ export default defineConfig({
 						},
 					],
 				},
+				{
+					label: 'Upgrade Guides',
+					autogenerate: {
+						directory: 'docs/upgrade-guides',
+						collapsed: true,
+					},
+				},
 				{
 					label: 'Components',
 					autogenerate: {
 						directory: 'docs/components',
 						collapsed: true,
 					},
 				},
+				{
+					label: 'Utilities',
+					autogenerate: {
+						directory: 'docs/utilities',
+						collapsed: true,
+					},
+				},
 			],
-			plugins: [starlightImageZoom()],
 		}),
+		// ui(),
```

Comment:
Is there a reason this is commented out? Might be worth removing this (with the import) or putting a comment on with a todo or something!

Reply:
Yup, this can be removed entirely. It has something to do with our setup specifically, just don't remember what exactly the reason was. Think it was because of Starlight, although I am not sure anymore.

Reply:
Removed for now!